frontend/src/components/Candidate-Decider/CandidateDecider.tsx

```diff
@@ -86,6 +109,21 @@ const CandidateDecider: React.FC<CandidateDeciderProps> = ({ uuid }) => {
     });
   };
 
+  const confirmNavigation = (direction: 'next' | 'previous' | 'other') => {
+    setIsOpen(false);
+    setIsBypassingModal(true);
+    setTimeout(() => {
```

Comment:
hm, wondering what the point of `setTimeout` with 0 milliseconds is?

Reply:
Oh oops I forgot to get rid of that - I was trying to fix a problem I was running into where it was making the user click save twice before it worked but I ended up fixing it a different way